### ISSUE TITLE
groups/talk: clicking "new messages" banner, or associated "x" button does not work sometimes

### ISSUE THREAD

**Issue by urcades** [tloncorp/landscape-apps#1461]
**Describe the bug, optionally with screenshots**
While browsing a talk channel with unread messages (new messages outside of the viewport), I often cannot successfully initiate the "view more messages" or "mark as read" actions in the banner. 

In the video below, I demonstrate the inconsistent nature of the unread banner — At 8 seconds, I manage to successfully clear the unreads of a channel. At 16 seconds I'm unable to mark another channel in the same group as read, or scroll to the first message.

https://user-images.githubusercontent.com/1195363/207400795-bbb1b82f-d546-4867-86b4-e9fa922d2e9d.mov

**To Reproduce**
1. This behavior is inconsistent in nature, but I'd go ahead and try to visit multiple groups/channels with long scrollbacks to test out which ones work and which ones don't

**Expected behavior**
Essentially, I expect the behavior of the unread banner to be deterministic/mechanical in nature, it should always work as labeled on the tin:
1. if I click the banner to view unread messages, I should be scrolled back into history with the earliest message displayed at the top of the scroll, and the banner should disappear
2. If I click on the "X" to clear unreads and ignore unread messages overflow, I should remain where I am in the scroll and the banner should disappear.

**Additional context:**
- Ship: planet (~tonlur-hidlyn)
- Instance type: Self-hosted (Digital Ocean)

Accessing the above urbit through the following machine, remotely:
- MacOS Ventura — 13.0.1 (22A400)
- Safari — Version 16.1 (18614.2.9.1.12)

**Comment by urcades** [comment]
A related experiential issue I can clearly define in another issue if necessary: 

When the "View Unreads" button is clicked and it successfully triggers the expected interaction, there's a small issue with the "Unread Messages blue demarcating line" component disappearing immediately, which can destabilize one's understanding of what the first unread message is.

We should update this interaction such that the small demarcating line that marks the first unread message **persists** under the following conditions:

1. The unread banner is visible and the channel is focused/being viewed by the user
2. The unread banner's "**View unread messages**" button has been clicked and the channel is focused/being viewed by the user
3. The unread banner's "**X**" button has been clicked and the channel is focused/being viewed by the user

The "Unread messages" line should disappear/un-persist/unload under the following conditions:

1. The unread banner has been clicked (Marked as Read OR Dismissed), the banner has disappeared accordingly, and the user has left the channel

~

In a nutshell, the conditions I'm listing above serve to illustrate that we need to decouple the in-chat "new messages demarcating line" interface from **the banner's visibility and unread state of the sidebar**, which have coupled state.

cc @jamesacklin let me know if this written spec is generally clear — I can actually illustrate/diagram it, but I'm trying to keep this issue spare.

**Comment by patosullivan** [comment]
This should have been fixed with #1490
